tests/e2e/test_easylink_run.py

```diff
@@ -12,7 +12,7 @@
 from easylink.utilities.general_utils import is_on_slurm
 from tests.conftest import RESULTS_DIR, SPECIFICATIONS_DIR
 
-RESULT_CHECKSUM = "9f9cc43beef9e688d809ff4f1cc9d569c51a276fb6dd277fa7d6ca5d57f81eb0"
+RESULT_CHECKSUM = "9cf076f863b2ee73a11da982240f977a708d5ea87dd28e435afaa92790824e52"
```

Comment:
we use secondary inputs in one of the implementations now, so the checksum has to change.